Bench-Kitura-TechEmpower/build.sh

```diff
@@ -31,12 +31,30 @@ Darwin)
   ;;
 esac
 
-# Build 'new' version
+# Build 'new' and "nio" version
 # If $REPO and $NEW_COMMIT are set, edit package $REPO to $NEW_COMMIT before building
+
+function cloneProject {
+  local projectPath=$1
+  local newPath=$2
+  pushd $projectPath
+  swift package reset
+  popd
+  if [ -d $newPath ]; then
+    preserveDir $newPath
+  fi
+  cp -R -p $projectPath $newPath
+}
+
 pushd $dir/latest/
 build "$dir/newBuild" "$REPO" "$NEW_COMMIT"
 popd
 
+cloneProject $dir/latest $dir/latest-nio
+pushd $dir/latest-nio/
+build "$dir/nio" "$REPO" "$NEW_COMMIT" "KITURA_NIO"
```

Comment:
Is it really necessary to change the `build` function?  Could we not just `export KITURA_NIO=1` before we invoke it (and `unset` it afterwards)?